tutorials/introduction.md

```diff
@@ -1,7 +1,131 @@
+- Add “Connect Wallet” button to your website [tutorial from ThirdWeb](https://blog.thirdweb.com/guides/add-connectwallet-to-your-website/)
+
+
+## 3. Define Token Gating Rules
+These are the rules that the wallet assets will be checked against by the Collab.Land API. The API will return boolean `True`/`False` if the account address fulfills the defined rules or not.
+
+#### Token Gating Rule (TGR) example
+
+```json!
+"rules": [
+    {
+      "type": "ERC721",
+      "chainId": 1,
+      "minToken": "1",
+      "contractAddress": "0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d",
+      "roleId": "001"
+    }
+  ]
+```
+The rule above indicates that  at least one ERC721 token of contract address `0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d` must be held in the user's wallet address.
+
+⚠ @Agnes - info needed on passing TGR definitions with API call
+- need info for how to construct TGRs and feed into the collab API
+- data schema json (could be an api link -- I'm not sure which)
+- docs could be collected on https://dev.collab.land/docs/local-development-setup/configuring-tgr
+
+
+## 4. Call the Collab.Land API
+Full documentation of the Collab.Land API can be [found here](https://dev.collab.land/docs/downstream-integrations/api/).
+
+
+⚠ I don't know how to interpret the api.collab.land site this page links to
+- POST request example [API reference](https://dev.collab.land/docs/downstream-integrations/api/token-gating#sample-request)
+    - PoC example showing data schema
+
+
+## 5. Handle API response
+There are several response options available with the Collab.Land API. Select which method you would like to use.
+
+### Sync with polling
+https://dev.collab.land/docs/downstream-integrations/api/token-gating#check-roles-synchronously
+- receive `True`/`False` from API 
+
+### Async via callback URI
+https://dev.collab.land/docs/downstream-integrations/api/token-gating#check-roles-asynchronously
+- receive callback URL and a notification via webhook at that URL when API data is available
+
+### Async via callback with polling
+https://dev.collab.land/docs/downstream-integrations/api/token-gating/#check-roles-asynchronously-and-poll-for-the-result
+- receive a `requestId` from API and poll for result
+
+## 6. Render gated content based on API response
+Display page content or new pages based on the `True`/`False` response of the Collab.Land API.
+
+### React example code
+This code snippet displays a "Hello World!" text popup when the Collab.Land API returns `True`.
+
+```json
+json code goes here
+```
+
+⚠ @Agnes to refactor POC site to simplify
+- example POC React snippet 
+    - text "hello World!" popup
+
+# Example Site Proof of Concept
+Follow along with the [proof of concept](https://member.collab.land/) site.
+
+⚠ @Agnes change POC site to only use frontend [wallet connect button](#22-Use-any-frontend-to-obtain-a-user%E2%80%99s-wallet-address) login
```

Comment:
@agnes512 change PoC site to only use Wallet Connect button, not collabland login